.changeset/honest-poets-walk.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/playground-ui': patch
+---
+
+Added an optional `leading` slot to `ToolCallPresentedHeader` for content rendered ahead of the tool icon, such as a timestamp.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)